src/utils/validation.ts

```diff
@@ -59,7 +65,8 @@ export const airkeeperDeploymentSchema = z.object({
 });
 
 export const deploymentSetSchema = z.object({
-  airnode: airnodeDeploymentSchema,
+  airnodeAWS: airnodeDeploymentSchemaAWS,
+  airnodeGCP: airnodeDeploymentSchemaGCP.optional(),
```

Comment:
To be honest I would have preferred this folder structure:
1. `/data/<provider>/deployments/<date>/airnode/aws`
2. `/data/<provider>/deployments/<date>/airnode/gcp`

but since this might just be my personal preference I would like to ask @aquarat what he thinks.

Reply:
that would probably be better, ill go ahead and make that change